Functions/Assertions/HaveParameter.Tests.ps1

```diff
@@ -85,30 +87,30 @@ InModuleScope Pester {
     Describe "Should -HaveParameter" {
 
         It "passes if the parameter <ParameterName> exists" -TestCases @(
-            @{ParameterName = "MandatoryParam"}
-            @{ParameterName = "ParamWithNotNullOrEmptyValidation"}
-            @{ParameterName = "ParamWithScriptValidation"}
+            @{ParameterName = "MandatoryParam" }
```

Comment:
I was talking about this, but the formatter does not add the space on the start line. And the many cases below.

Reply:
But running the automatic formatter on my side adds a lot of undesired changes. Maybe it is because I am running on preview IDK. Merging as is.